contributed/ra_dec_dc2_run2.2i_dask.ipynb

```diff
@@ -0,0 +1,1455 @@
+    "    major_ticks_ra = np.linspace(ra_range[0], ra_range[1], n_ra)\n",
+    "    major_ticks_dec = np.linspace(dec_range[0], dec_range[1], n_dec)\n",
+    "    # If you set the Dec to the be the rotation center you get Delta x steps that are constant\n",
+    "    left_ra = np.zeros(n_dec) + dc2_run2x_wfd_df['ra'][0]  # It doesn't matter what this is, because it doesn't affect Dec.\n",
+    "    bottom_dec = np.zeros(n_ra) + dc2_run2x_wfd_df['dec'][2]\n",
+    "    major_ticks_x, _ = mollweide.ang2xy(major_ticks_ra, bottom_dec, lonlat=True)\n",
+    "    # RA doesn't matter for Dec\n",
+    "    _, major_ticks_y = mollweide.ang2xy(left_ra, major_ticks_dec, lonlat=True)\n",
+    "\n",
+    "    major_ticks_and_labels_x = [(x, f\"{ra:0.0f}\") for x, ra in zip(major_ticks_x, major_ticks_ra)]\n",
+    "    major_ticks_and_labels_y = [(y, f\"{dec:0.0f}\") for y, dec in zip(major_ticks_y, major_ticks_dec)]\n",
+    "    \n",
+    "    return major_ticks_and_labels_x, major_ticks_and_labels_y\n",
+    "\n",
+    "\n",
+    "def decorate_ra_dec_plot(ra_dec_plot, mollweide=mollweide):\n",
+    "    \"\"\"\"\n",
+    "    Set up the color bar, color range, clipping and color map and set the size of our plot\n",
+    "    \"\"\"\n",
+    "    major_ticks_and_labels_x, major_ticks_and_labels_y = generate_ra_dec_tick_labels(mollweide)\n",
+    "    ra_dec_plot = ra_dec_plot.opts(xlabel='RA', ylabel='Dec', xticks=major_ticks_and_labels_x, yticks=major_ticks_and_labels_y)\n",
+    "    ra_dec_plot = ra_dec_plot.opts(hv.opts.Image(colorbar=True, clim=(10, None), clipping_colors={'min': 'gray'},\n",
+    "                                   cmap=process_cmap(\"viridis\", provider=\"matplotlib\")))\n",
+    "    ra_dec_plot = ra_dec_plot.opts(width=480, height=400)\n",
+    "\n",
+    "    return ra_dec_plot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ra_dec_moll = decorate_ra_dec_plot(ra_dec_moll, mollweide)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dc2_ra_dec_coverage = ra_dec_moll * wfd_outline * ddf_outline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dc2_ra_dec_coverage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hv.save(dc2_ra_dec_coverage, \"dc2_ra_dec_coverage.html\", backend=\"bokeh\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Cool!  We did the projection, it ran across our workers and we can even zoom in/and out and it will dynamically rebin.\n",
+    "\n",
+    "But watch what happens with the Dask Dashboard when you zoom in and out.\n",
+    "\n",
```

Comment:
Actually for me, I saw nothing happen on the dashboard, and wasn't even sure it was redoing anything.  I realized later this was described above as not working in the juptyerlab environment, so I would change this text to say 'if...' or explain why they might not be seeing it.